packages/plugins/whisper-geth/src/whisperGethClient.js

```diff
@@ -157,6 +156,7 @@ class WhisperGethClient {
     let rpcApi = this.config.rpcApi;
     let wsApi = this.config.wsApi;
     let args = ['--ipcdisable']; // Add --ipcdisable as ipc is not needed for Whisper and it conflicts on Windows with the blockchain node
+    args.push("--port=30304");
```

Comment:
I think you said in your other PR that the port number was not used for Whisper. I think thins would be a good opportunity to fix it.
Something like this:
```js
args.push(`--port="${this.communicationConfig || 30304}"`);
```